docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+
+## Create a Profile
+
+VS Code treats your current configuration as the **Default Profile**. As you modify settings, install extensions, or open and close views and panels, these customizations are tracked in the Default Profile.
+
+To create a new profile, you can use the **File** > **Preferences** > **Profiles** > **Create Profile** menu item (**Code** > > **Preferences** > **Profiles** > **Create Profile** on macOS).
+
+![Create Profile command](images/profiles/create-profile.png)
+
+You can create a new profile based on the current profile (**Profiles: Create from Current Profiles**) or create an Empty Profile. An Empty Profile includes no user customizations (settings, extensions, snippets, etc.) but with the UI state (view layouts and sizing) preserved.
+
+You can also access the Profile command menu via the **Manage** gear button in the lower right of the Activity bar.
+
+![Create Profile command via the Manage button in the Activity bar](images/profiles/create-profile-via-manage.png)
+
+The **Profiles: Create Profile** command is also available in the Command Palette (`kb(workbench.action.showCommands)`).
+
+Once you choose whether to create a new profile based on the current profile or an empty profile, you are prompted to enter a name for the new profile.
+
+### Check the current profile
+
+The current profile name is displayed in several places in the VS Code UI:
+
+* Title bar
+* **File** > **Preferences** > **Profiles**
+* **Manage** gear button hover
+
+If you are still using the Default Profile, no profile name is displayed.
+
+The **Manage** gear button displays a badge with the first two letters of the active profile so you can quickly check which profile you are running.
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
+
+### Workspace associations
+
+When you select create or select an existing profile, it is associated with the current workspace and whenever you open that folder, the workspace's profile is active. If you open another folder, the profile either goes back to the Default Profile if no profile is set, or switches to that folder's associated profile.
+
+Setting a workspace's profile back to the Default Profile clears the workspace's profile association.
+
+## Managing profiles
+
+### Switch profiles
+
+You can quickly switch between profiles with the **Profiles: Switch Profile** command in the Command Palette, which presents a dropdown listing your available profiles.
```

Comment:
Should we mention here "through the profiles menu or ...." ?

Reply:
I figured people had already discovered the profile list in the profile menus and the check UI. This section is for discovery of the quick switching command.

Reply:
Makes sense. Should we then say "You can also switch"?